package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "concord",
-  "version": "0.2.1",
+  "version": "0.3.1",
```

Comment:
Better make it a patch version. Doesn't break API.

Reply:
this is deliberate. New functionality, even when non breaking, is a minor update, not patch, according to https://semver.org/